client/src/frontend/components/resource-slide/SlideImage.js

```diff
@@ -1,38 +1,28 @@
 import React, { Component } from "react";
 import PropTypes from "prop-types";
-import ResourcePreview from "frontend/components/resource-preview";
-import IconComposer from "global/components/utility/IconComposer";
+import get from "lodash/get";
+import { Zoom } from "./Slide";
 
 export default class ResourceListSlideFigureImage extends Component {
-  static propTypes = {
-    resource: PropTypes.object,
-    enableZoom: PropTypes.bool
-  };
+  static displayName = "ResourceList.Slide.Image";
 
-  static defaultProps = {
-    enableZoom: true
+  static propTypes = {
+    resource: PropTypes.object.isRequired
   };
 
   render() {
-    const attr = this.props.resource.attributes;
+    const background = get(
+      this.props,
+      "resource.attributes.attachmentStyles.medium"
```

Comment:
As I've worked on manifold more, I've come to think that using lodash get is wasteful because it has to do string parsing to traverse the object.

I'd do this instead:

```
get background() {
  const { resource } = this.props;
  if (!resource.attributes.attachmentStyles) return null;
  return resource.attributes.attachmentStyles.medium;
}